src/components/Calculator.js

```diff
@@ -1,48 +1,46 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
-/* eslint-disable */
 
-class Calculator extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {total: 0, next: null, operation: null}
-  }
+function handleClick(event, state, setter) {
+  const buttonName = event.target.innerText;
+  const result = calculate(state, buttonName);
+  setter(result);
+}
```

Comment:
- [ ] Please use Es6 arrow functions to define the `handleClick` function.
- [ ] [OPTIONAL] Instead of having the `handleClick` function outside the `Calculator` component, please consider placing it inside as the `handleClick` defines what should happen in the component and ideally should not be outside its definition.